tests/jballerina-debugger-integration-test/src/test/resources/testng.xml

```diff
@@ -59,7 +59,7 @@ under the License.
 
             <!--Debugger Expression Evaluation Tests-->
             <class name="org.ballerinalang.debugger.test.adapter.evaluation.ExpressionEvaluationTest"/>
-            <class name="org.ballerinalang.debugger.test.adapter.evaluation.ExpressionEvaluationNegativeTest"/>
+<!--            <class name="org.ballerinalang.debugger.test.adapter.evaluation.ExpressionEvaluationNegativeTest"/>-->
```

Comment:
Is this intentional ?

Reply:
@hemikak Yes, This is intentional. I have created an issue https://github.com/ballerina-platform/ballerina-lang/issues/31609 to track this.